partner_credit_limit/model/partner.py

```diff
@@ -40,7 +41,10 @@ def _get_credit_overloaded(self):
 
     @api.model
     def movelines_domain(self, partner):
+        """Return the domain for search the
+        account.move.line for the user's company."""
         domain = [('partner_id', '=', partner.id),
+                  ('company_id', '=', self.env.user.company_id.id),
```

Comment:
Hi @JulioSerna 

Yes, a customer and I tested it without this line. Actually, the client requested this change and @luistorresm advised me to do it in this repo and thus have it available for other clients.
Here is a small video with my test https://youtu.be/d9p8Hs4JkiQ

Thanks!

Reply:
@edy1192 

Did you test it with another customer DB?

I'm wonder if will be needed a migration script, in order to set old values on new properties

Reply:
Hi @hugho-ad I tested it here https://fertico-11-0-partner-credit-limit-edy1192-634184.dev.odoo.com/web this is a customer db.  Yesterday I sent a video https://youtu.be/7Qyi-SSNMt8 to the customer with my tests. 

With this video you can know if the script is necessary? I ask why I don't understand what you mean.

I await your comments

Thanks in advance.

Reply:
@edy1192 

Since fields now will become on property fields, I don't know id the value was set on it

I mean:
e.g.
Before:
partner.credit_limit  return 5 from res.partner table

After
partner.credit_limit returns ??  not sure it will return the original value 5 
or False since this is now get it from ir.properties

Regards!

Reply:
Hello @hugho-ad 

These fields are computed with store=False, why you think that the value not will be updated?

Regards

Reply:
Hi @hugho-ad 

I tested what you told me, https://youtu.be/z6quTSznYpA 

Like like you said, the credit limit value is lost after the change.

The migration script you propose what should It do?

Thanks in advance
cc @luistorresm

Reply:
@luistorresm tests made by @edy1192  proved what I said

@edy1192 
the migration script should recover all that values and set them again in the ir.properties for each partner

values must exist after the upgrade, so you will create a post-migration script that will read the columns from res.partner table and then write the new value on the ir.property

something like following:

```sql 
select id, credit_limit, from res_partner where credit_limit is not null;
```

```python
# then you read the result and make id browseables records
data = res.dictfetchall()
partners = env['res.parnter'].browse(res.get('ids') or [])
for partner in partners:
    # not sure, if is needed here pass the company_id
    partner.write({'credit_limit': data[parner.id]})  # This will automatically set the ir.property
```

Note: The above is pseudo code, just to show you the main idea about the job to be done

Regards!

Reply:
@edy1192 

you tested with admin, admin pass the ir.rule

Reply:
Good point @JulioSerna Let me try with normal users. I'll let you know the results.

Reply:
@JulioSerna @hugho-ad I tried with demo user and here is the result https://youtu.be/LeavM1fmDJ8

Could you let me know your comments?

Regards.

Reply:
@edy1192 

LGTM 👍

Reply:
@moylop260 here is the approval